pkg/clients/rds/rds.go

```diff
@@ -107,8 +109,6 @@ func GenerateCreateDBInstanceInput(name, password string, p *v1alpha2.RDSInstanc
 		PromotionTier:                      awsclients.Int64Address(p.PromotionTier),
 		PubliclyAccessible:                 p.PubliclyAccessible,
 		StorageEncrypted:                   p.StorageEncrypted,
-		TdeCredentialArn:                   p.TDECredentialArn,
-		TdeCredentialPassword:              p.TDECredentialPassword,
```

Comment:
Any context on removing these guys? I'm guessing the latter is a credential?

Reply:
>  I'm guessing the latter is a credential?

Right. I've left the following comment in `rdsinstance_types.go`:
```
// TODO(muvaf): get this password as input when we have a way of supplying
// sensitive information as input from the user.

// TDECredentialArn is the ARN from the key store with which to associate the instance for TDE encryption.
// +optional
// TDECredentialArn *string `json:"tdeCredentialArn,omitempty"`
// TDECredentialPassword is the password for the given ARN from the key store in order to access the
// device.
// +optional
// TDECredentialPassword *string `json:"tdeCredentialPassword,omitempty"`
```

As far as I understand, `TDECredentialArn` has no meaning if you don't supply the password. So, I've removed both.